lua/nvim-treesitter/parsers.lua

```diff
@@ -2650,6 +2650,15 @@ list.templ = {
   maintainers = { "@vrischmann" },
 }
 
+list.circom = {
+  install_info = {
+    url = "https://github.com/Decurity/tree-sitter-circom.git",
+    files = { "src/parser.c" },
+    branch = "main",
```

Comment:
```suggestion
```

Only needed if it's not the default branch.

And please make sure that 

1. `tree-sitter build`
2. `tree-sitter generate`

works without errors and warnings.

Reply:
ok

Reply:
after removing branch = "main" below error occurred

```
mv: cannot stat 'tree-sitter-circom-tmp/tree-sitter-circom-master': No such file or directory
Error detected while processing command line:
Failed to execute the following command:
{
  cmd = "mv",
  opts = {
    args = { "-f", "tree-sitter-circom-tmp/tree-sitter-circom-master", "tree-sitter-circom" },
    cwd = "/home/runner/.local/share/nvim"
  }
}
```

Reply:
@clason

Reply:
You need to add a lockfile.json entry for the new parser, too (again, keeping the list alphabetical).